src/node.cc

```diff
@@ -3925,6 +3906,12 @@ static void ParseArgs(int* argc,
     } else if (strcmp(arg, "--") == 0) {
       index += 1;
       break;
+    } else if (strcmp(arg, "--abort-on-uncaught-exception") ||
+               strcmp(arg, "--abort_on_uncaught_exception")) {
```

Comment:
should be `strcmp(...)  == 0`:
```
>>>     CID 169223:  Integer handling issues  (CONSTANT_EXPRESSION_RESULT)
>>>     The "or" condition "strcmp(arg, "--abort-on-uncaught-exception") || strcmp(arg, "--abort_on_uncaught_exception")" will always be true because "arg" cannot be equal to two different values at the same time, so it must be not equal to at least one of them.
3909         } else if (strcmp(arg, "--abort-on-uncaught-exception") ||
3910                    strcmp(arg, "--abort_on_uncaught_exception")) {
3911           abort_on_uncaught_exception = true;
3912           // Also a V8 option.  Pass through as-is.
3913           new_v8_argv[new_v8_argc] = arg;
3914           new_v8_argc += 1;
```